multicast/env.py

```
@@ -91,12 +91,12 @@
 	from . import logging  # skipcq: PLY-C0414
 	from . import socket  # skipcq: PYL-C0414
 	import ipaddress
-except Exception as err:
-	baton = ImportError(err, str("[CWE-758] Module failed completely."))
+except ImportError as _cause:
+	baton = ImportError(_cause, str("[CWE-758] Module failed completely."))
```

Ruff (0.8.2)

Unnecessary `str` call (rewrite as a literal)

Replace with string literal

(UP018)